backend/src/v5/processors/teamspaces/projects/models/federations.js

```diff
@@ -83,4 +86,20 @@ Federations.getFederationStats = async (teamspace, federation) => {
 	};
 };
 
+Federations.updateSettings = async (teamspace, federation, payload) => {
+	if (payload.defaultView) {
```

Comment:
This is essentially identical to the container code right?

add a `settings.js` in the common folder and just call that?